src/gwt/src/org/rstudio/studio/client/workbench/ui/WorkbenchTabPanel.java

```diff
@@ -298,11 +303,17 @@ public LogicalWindow getParentWindow()
       return parentWindow_;
    }
 
+   public void neverVisible(boolean value)
```

Comment:
I'd change this to follow Java setter conventions, `setNeverVisible(boolean value)`

Reply:
Fixed!